src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/participationhistory/restapi/CourseParticipationIntegrationTest.kt

```diff
@@ -52,19 +46,18 @@ class CourseParticipationIntegrationTest : IntegrationTestBase() {
     val expectedCourseParticipation = CourseParticipation(
       id = createdCourseParticipationId,
       courseId = courseId,
-      prisonNumber = PRISON_NUMBER,
+      prisonNumber = prisonNumber,
     )
 
     retrievedCourseParticipation shouldBe expectedCourseParticipation
   }
 
-  @DirtiesContext
   @Test
   fun `Creating a course participation with a valid course id but a different course name should return 400 with error body`() {
     val createCourseParticipationErrorResponse = createCourseParticipation(
       courseId = getFirstCourseId(),
       otherCourseName = "A Course",
-      prisonNumber = PRISON_NUMBER,
+      prisonNumber = prisonNumber(),
```

Comment:
`randomPrisonNumber()` makes it a bit clearer that we're feeding one in directly here.